10_functions/toString.js

```diff
@@ -0,0 +1,6 @@
+var str = "hello everyone";
+
+function toString(){
+    return str.toString(str);
```

Comment:
I don't think this is doing what you expect it to be doing. What do you expect it to be doing?